tests/form/robot/ValidationTextBox.html

```diff
@@ -495,13 +495,39 @@
 							doh.robot.typeKeys('123', 1000, 600);
 							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
 								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
-								doh.is(undefined, this.textbox.get('value'), "get('value')");
+								doh.is(123, this.textbox.get('value'), "get('value')");
 								doh.f(this.textbox.isValid(), "!isValid()");
```

Comment:
If the editOptions solution works, should I update my change to just supply
an editOptions.places if constraints.places exists and editOpitons.places
does not exist?

On Fri, May 8, 2015 at 5:22 PM, Bill Keese <notifications@github.com> wrote:

> In tests/form/robot/ValidationTextBox.html
> <https://github.com/dojo/dijit/pull/93#discussion_r29981713>:
>
> > @@ -495,13 +495,39 @@
> >  							doh.robot.typeKeys('123', 1000, 600);
> >  							doh.robot.sequence(d.getTestCallback(dojo.hitch(this, function(){
> >  								doh.is('123', this.textbox.focusNode.value, "focusNode.value");
> > -								doh.is(undefined, this.textbox.get('value'), "get('value')");
> > +								doh.is(123, this.textbox.get('value'), "get('value')");
> >  								doh.f(this.textbox.isValid(), "!isValid()");
>
> PS: I notice though that with the markup below, isValid() is true even
> before the blur:
>
> <input type="text" data-dojo-type="dijit/form/NumberTextBox"
>     data-dojo-props="constraints:{places:2}, editOptions:{places:'0,2'}">
>
> So probably we should be consistent with that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dojo/dijit/pull/93/files#r29981713>.
>